llvm/test/CodeGen/X86/fp-strict-libcalls-msvc32.ll

```diff
@@ -177,6 +177,107 @@ define float @tan(float %x) #0 {
   ret float %result
 }
 
+define float @acos(float %x) #0 {
```

Comment:
@efriedma-quic I'm looking at the commit history of this file:
https://github.com/llvm/llvm-project/commits/release/19.x/llvm/test/CodeGen/X86/fp-strict-libcalls-msvc32.ll

The auto cherry pick failed because I added the test for msvc out of order in https://github.com/llvm/llvm-project/commit/378fe2fc23fa56181577d411fe6d51fa531cd860

That commit added some vectorizations nothing that should impact the fix.

Reply:
Okay that's fine.